apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/post-published-modal/index.tsx

```diff
@@ -28,6 +28,9 @@ const PostPublishedModal: React.FC = () => {
 	const { fetchShouldShowFirstPostPublishedModal, setShouldShowFirstPostPublishedModal } =
 		useDispatch( 'automattic/wpcom-welcome-guide' );
 
+	const { siteUrlOption, launchpadScreenOption } = window?.launchpadOptions || {};
+	const siteUrl = siteUrlOption.replace( 'http://', '' );
```

Comment:
Wondering if there's a way to get the site slug without the `http://` part

Reply:
> You can try using window.location.host

The truncated url we're trying to obtain isn't from the browser window, so calling `window.location.host` wouldn't return  what we're looking for.